e2e/tests/components/dashboard/dashboard.e2e-spec.ts

```diff
@@ -271,4 +271,53 @@ describe('Dashboard Tests', () => {
 
         expect(announcement).toBe('Moving and resizing cancelled. Dashboard with 4 columns, containing 4 panels. Usage Analytics panel in row 0, column 0, is 4 columns wide and 2 rows high. Service panel in row 2, column 0, is 2 columns wide and 1 rows high. Users panel in row 2, column 2, is 1 columns wide and 1 rows high. Alert panel in row 2, column 3, is 1 columns wide and 1 rows high. Press space to move and resize the Usage Analytics panel.');
     });
+
+    it('should allow the rowSpan to remain the same size in stacked mode as regular mode', async () => {
+        // resize page so in stacked mode
+        await browser.driver.manage().window().setSize(400, 600);
+
+        // check we are in grab mode
+        const widgetSpan = await widget1.$('div').getAttribute('class');
+
+        // rowSpan remains as 2 in stacked mode
+        expect(widgetSpan).toContain('widget-row-span-2');
+
+        expect(await imageCompare('dashboard-stacked-mode-rowSpan')).toEqual(0);

```

Comment:
Could you use the "full screen" version of imageCompare in order to capture the position and size of all four widgets?